vendor/engines/projects/app/controllers/projects/global_search_controller_extension.rb

```diff
@@ -0,0 +1,13 @@
+module Projects
+
+  module GlobalSearchControllerExtension
+
+    extend ActiveSupport::Concern
+
+    included do
+      searcher_classes.unshift(Projects::GlobalSearch::ProjectSearcher)
```

Comment:
What's your reasoning for doing an `unshift` instead of `<<`? I'm inclined towards orders always being top priority, but could possible be swayed.